src/util/queryBankBalances.ts

```diff
@@ -0,0 +1,23 @@
+import { QueryClient, createProtobufRpcClient } from '@cosmjs/stargate';
+import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
+import { QueryClientImpl } from 'cosmjs-types/cosmos/bank/v1beta1/query';
+import type { Coin } from '@cosmjs/stargate';
+import type { HttpEndpoint } from '@cosmjs/tendermint-rpc';
+
+// XXX use casting support when possible for load balancing, batching, and
```

Comment:
Do we have a ticket to track when it's possible? I think we should. Consider noting it here with an `UNTIL`.

Reply:
Not sure about the ticket yet, changed to `UNTIL`